node_modules/oae-tenants/config/config.js

```diff
@@ -91,12 +91,5 @@ module.exports = {
         'elements': {
             'timezone': new Fields.List('Tenant Timezone', 'Tenant timezone', 'Etc/UTC', timezoneConfigValues, {'suppress': true})
         }
-    },
```

Comment:
I was more thinking along the lines of the ConfigAPI trying to map data from the DB to fields that no longer exist